### ISSUE TITLE
Generating forms for TD from TM conversion

### ISSUE THREAD

**Issue by egekorkan** [eclipse/editdor#64]
- Minor improvement

When a TD is generated from a TM, it would be nice to generate at least some boilerplate forms objects with empty href and default content types

**Comment by BessemMnif** [comment]
Do you mean here if a user didn´t define a href and content types in the Thing Model then instead of generating an empty form in the TD, generates a form with an empty href and application/json as content types? If this is the case, can you give me a hint why you think that this is better than having an empty form?

**Comment by egekorkan** [comment]
Yes I meant that :) The reason was to write less to have a valid TD. `contentType` and `href` can both be empty. `op` can (and in my personal opinion should) be generated as well.
